.github/dependabot.yml

```diff
@@ -11,3 +11,7 @@ updates:
           - "> 2.1"
     commit-message:
       prefix: dependency
+  - package-ecosystem: github-actions
```

Comment:
Please add commit message prefix as we do in maven config above.

Reply:
Sure @nrmancuso

Reply:
@nrmancuso Added commit message.

Reply:
Done

Reply:
This will cause CI to fail, we do not allow this commit prefix. @romani @rnveach are we good with just using `dependency` prefix for actions dependencies too? I am.

Reply:
I am fine with `dependency` being used for anything representing an external version upgrade.

Reply:
Ok, @AayushSaini101 lets just reuse `dependency` prefix, after all, it is “dependabot”

Reply:
Done @nrmancuso

Reply:
@AayushSaini101 , please do not resolve conversations, it should b done by maintainers